lib/single_cov.rb

```diff
@@ -76,6 +76,23 @@ def assert_tested(files: glob('{app,lib}/**/*.rb'), tests: default_tests, untest
       end
     end
 
+    def refute_complete_coverage_drop(tests: default_tests, currently_complete: [], currently_complete_file: "")
+      complete = tests.select { |file| File.read(file) =~ /SingleCov.covered!(\s*|\s*\#.*)$/ }
+      missing_complete = currently_complete - complete
+
+      raise "The following file(s) was at 100% SingleCov test coverage and is no longer. " +
```

Comment:
also best spell it out so it's clear:
```suggestion
      raise "The following file(s) were marked as 100% SingleCov test coverage (had no `coverage:` option) and are no longer. " +
```

Reply:
Good suggestion. Thank you